### PR TITLE
[Preview] add the signature parsing.

### PR DESCRIPTION
**Pull request by larry-aptos**
* This change adds parsing support for new signature types.

### CODE REVIEW

File: rust/processor/src/models/user_transactions_models/signatures.rs [hunk 1]
```diff
@@ -88,8 +96,21 @@ impl Signature {
             SignatureEnum::MultiEd25519(_) => String::from("multi_ed25519_signature"),
             SignatureEnum::MultiAgent(_) => String::from("multi_agent_signature"),
             SignatureEnum::FeePayer(_) => String::from("fee_payer_signature"),
-            SignatureEnum::SingleSender(_sender) => {
-                panic!("SingleSender signature is not supported")
+            SignatureEnum::SingleSender(sender) => {
```
Comment (banool) [inline rust/processor/src/models/user_transactions_models/signatures.rs:99]: This sort of looks like code dupe from above, is there a way to pull this out into a common function? 

At the least we should probs use constants for the strings.

File: rust/processor/src/models/user_transactions_models/signatures.rs [hunk 2]
```diff
@@ -281,12 +302,173 @@ impl Signature {
...
+    fn parse_single_key_signature(
+        s: &SingleKeySignaturePb,
+        sender: &String,
+        transaction_version: i64,
+        transaction_block_height: i64,
+        is_sender_primary: bool,
+        multi_agent_index: i64,
+        override_address: Option<&String>,
+    ) -> Self {
+        let signer = standardize_address(override_address.unwrap_or(sender));
```
Comment (banool) [inline rust/processor/src/models/user_transactions_models/signatures.rs:337]: Same here re code dupe.

File: rust/processor/src/models/user_transactions_models/signatures.rs [hunk 3]
```diff
@@ -281,12 +302,173 @@ impl Signature {
...
+        let signature = s.signature.as_ref().unwrap();
+        let type_ = match AnySignatureTypeEnumPb::try_from(signature.r#type) {
+            Ok(AnySignatureTypeEnumPb::Ed25519) => String::from("single_key_ed25519_signature"),
+            Ok(AnySignatureTypeEnumPb::Secp256k1Ecdsa) => {
+                String::from("single_key_secp256k1_ecdsa_signature")
             },
-            AccountSignatureEnum::MultiKeySignature(_sig) => {
-                panic!("MultiKeySignature is not supported")
+            _ => {
+                panic!("Unspecified signature type or un-recognized type is not supported")
```
Comment (banool) [inline rust/processor/src/models/user_transactions_models/signatures.rs:345]:
```suggestion
            wildcard => {
                panic!("Unspecified signature type or un-recognized type is not supported: {:?}", wildcard)
```

**Comment by larry-aptos** [comment]
I don't have time now to refactor this... 

i am going to merge this and do post-submission follow-up.

thank you Ser

**Pull request merged**